app/helpers/spree/admin/webhooks_subscribers_helper.rb

```diff
@@ -0,0 +1,42 @@
+module Spree
+  module Admin
+    module WebhooksSubscribersHelper
+      ADDITIONAL_EVENTS_LISTS = {
```

Comment:
@aleksandarpetrushev let's move this code into the `Subscriber` model as `LIST_OF_ALL_EVENTS` constant so we have one source of truth for these